cirq-core/cirq/work/sampler.py

```diff
@@ -60,6 +70,10 @@ async def run_async(
     ) -> 'cirq.Result':
         """Asynchronously samples from the given Circuit.
 
+        Provides measurement outcomes as a `Result` object.  This
```

Comment:
maybe `cirq.Result` in this one so it links?

Reply:
Done.